extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeCommentsInfoItemExtractor.java

```diff
@@ -59,13 +60,15 @@ public DateWrapper getUploadDate() throws ParsingException {
     }
 
     @Override
-    public String getCommentText() throws ParsingException {
+    public Description getCommentText() throws ParsingException {
         final String htmlText = JsonUtils.getString(item, "text");
```

Comment:
Since here the comment is returned as HTML, can't you just return `Description(htmlText, Description.HTML)`?

Reply:
No, because it contains some junk like css classes, etc.